apps/web-mzima-client/src/app/shared/components/collections/collections.component.html

```diff
@@ -89,6 +89,8 @@
           [checked]="isPostInCollection(item)"
           (checkedChange)="onCheckChange($event, item)"
           [data-qa]="'collection-item'"
+          tabindex="0"
+          (keydown)="onKeydown($event, item)"
```

Comment:
Now to the review comment for the PR @kivuvarosekivuvan 
Since clicking on the collection cards takes you to some page and closes the modal, logically, it feels like it should be a link. 

Here's my suggestion: Maybe you want to go into/inside the app-collection-item component's html and wrap the code in there with a link element instead.

Plus we'll be able to avoid the use of `tabIndex` that way, and also we don't need any or so much javascript for it to cooperate.

And then that it's a link may even also be good for tools like screen readers. since it will announce it that it's a link to some other page.